devito/ir/support/space.py

```diff
@@ -523,7 +523,14 @@ def project(self, cond):
         else:
             func = lambda i: i in cond
         intervals = [i for i in self.intervals if func(i.dim)]
-        sub_iterators = {k: v for k, v in self.sub_iterators.items() if func(k)}
+        sub_iterators = {}
+        for k, v in self.sub_iterators.items():
```

Comment:
what was the issue here?

Reply:
Some non-modulo dimension where ending up in the for loop signature crashing openMP simd

Reply:
OK, reminder: I need to look into this.

What's a test causing the issue w/o this fix?